llvm/unittests/Analysis/VectorFunctionABITest.cpp

```diff
@@ -103,6 +94,12 @@ class VFABIParserTest : public ::testing::Test {
     return NGlobalPreds == 1 && Info.Shape.Parameters.back().ParamKind ==
                                     VFParamKind::GlobalPredicate;
   }
+
+  // Checks that the number of vectorized parameters matches the scalar ones.
+  // Takes into account that vectorized calls may also have a Mask.
```

Comment:
could be worth to add that it requires correct ScalarFunction type to be used in the "invokeParser"